tests/backup_role/test_backup.yml

```diff
@@ -383,6 +383,31 @@
       loop_var: server_backup_data
       label: server_backup_data.path
 
+  - name: Test issue 900 fix.
+    block:
+      - name: Remove all backup from server.
+        ansible.builtin.include_role:
+          name: ipabackup
+        vars:
+          state: absent
+          ipabackup_name: all
```

Comment:
This task should not be in the block. The rescue should only be for the next task.